service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java

```diff
@@ -179,14 +181,15 @@ private String buildDoseQuantityPertinentInformation(Immunization immunization)
     }
 
     private String buildNotePertinentInformation(Immunization immunization) {
-        String notes = StringUtils.EMPTY;
-        if (immunization.hasNote()) {
-            List<Annotation> annotations = immunization.getNote();
-            notes = annotations.stream()
-                .map(Annotation::getText)
-                .collect(Collectors.joining(StringUtils.SPACE));
-        }
-        return notes;
+        return Stream.concat(
+            immunization.hasNote() ? immunization.getNote().stream() : Stream.empty(),
+            messageContext.getInputBundleHolder().getRelatedConditions(immunization.getId())
```

Comment:
The story says to append the additional text, but all of out other mappings prepend to pertinent information. Could you confirm with @dmcavenue1 which one it should be?